src/qililab/qprogram/experiment_executor.py

```diff
@@ -104,10 +106,13 @@ class ExperimentExecutor:
         - The results will be saved in a timestamped directory within the `base_data_path`.
     """
 
-    def __init__(self, platform: "Platform", experiment: Experiment):
+    def __init__(self, platform: "Platform", experiment: Experiment, calibration: Calibration | None):
         self.platform = platform
         self.experiment = experiment
 
+        if calibration and calibration.crosstalk_matrix:
+            self.platform.crosstalk = calibration.crosstalk_matrix
+
```

Comment:
Last thing, let's remove this for now, since platform already has the crosstalk_matrix, and if users want to change now have the `experiment.set_crosstalk()` method.

In general, i want us to have the `Calibration` passed to the `execute()` method, but we have to think how to propagate it to  the whole experiment, like the `execute_qprograms` and so on, so let's do it in another PR after some discussion.